openedx_webhooks/github/handlers/bouncer/__init__.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+import logging
+
+from ....services.rq import q as _q
+from ...models import GithubHeader
+from .rules import RULES
+
+logger = logging.getLogger(__name__)
```

Comment:
We can definitely remove it for now. My intention was to leave them there and go back to set up logging if we can agree on this design pattern. But we can definitely remove now and add logger in later.